core/src/main/scala/chisel3/properties/Property.scala

```diff
@@ -23,7 +23,11 @@ private[chisel3] class PropertyType[T]
   * Typeclass instances for valid Property types are defined here, so they will
   * be in the implicit scope and available for users.
   */
-private[chisel3] object PropertyType {}
+private[chisel3] object PropertyType {
+  implicit val intPropertyTypeInstance = new PropertyType[Int]
```

Comment:
Is it necessary to have properties for these various integer types as opposed to just a single integer property type (such as `BigInt`)?

Reply:
It's a good question. My thought is users might have either an Int, a Long, or a BigInt in their code, any of which can be converted to a FIRRTL Integer as it is being specced out. So I went ahead and added typeclass instances here for all of the above, so users can make a compatible property out of whatever Scala integer they happen to have in hand. If we only supported BigInt at this level, and a user had an Int, they'd have to make it into a BigInt before they could use it in a property. I don't think that's a huge inconvenience, but it's also pretty simple to ease that minor inconvenience.